### PR TITLE
Fix build on circle CI

### PR DESCRIPTION
**Pull request by Undermaken**
This PR aims to fix isseus about circle CI build

### CODE REVIEW

File: src/utils/azure.ts [hunk 1]
```diff
@@ -42,3 +42,21 @@ export const getStorageConnection = async (name: string) =>
...
     `az storage account show-connection-string --name ${name} --output tsv`
   )).stdout;
+
+/**
+ * hasCosmosConnection checks if the host has az cli installed and resouceGroup and name
+ * are valid inputs for cosmosdb
+ */
+export const hasCosmosConnection = (resourceGroup: string, name: string) => {
+  try {
+    execa.sync(
```
Comment (cloudify) [inline src/utils/azure.ts]: use the async version

File: src/utils/azure.ts [hunk 2]
```diff
@@ -42,3 +42,21 @@ export const getStorageConnection = async (name: string) =>
...
+      `az cosmosdb show -g ${resourceGroup} -n ${name} --query documentEndpoint -o tsv`
+    );
```
Comment (cloudify) [inline src/utils/azure.ts]: perhaps you should check the output instead of blindly returning `true`?

File: test/commands/users/list.test.ts [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
+describe("list all profiles", () => {
+  const isCosmosConnectionAvailable = hasCosmosConnection(
+    "agid-rg-test",
+    "agid-cosmosdb-test"
+  );
+  /**
+   * if host has the az client installed and valid cosmos credentials we test command first output line matches
+   * the header row (fiscalCode column name). Otherwise the test expects for an error with 2 as exit status
+   */
+  if (isCosmosConnectionAvailable) {
```
Comment (cloudify) [inline test/commands/users/list.test.ts]: these should be two different tests - you can follow [this pattern](https://github.com/facebook/jest/issues/7245#issuecomment-440774078) to skip tests that require `az`

File: src/utils/azure.ts [hunk 4]
```diff
@@ -42,3 +42,29 @@ export const getStorageConnection = async (name: string) =>
...
+    const primaryReadonlyMasterKey = await execa(
+      `az cosmosdb list-keys -g ${resourceGroup} -n ${name} --query primaryReadonlyMasterKey -o tsv`
+    );
+
+    return (
+      documentEndpoint.exitCodeName === successExitCode &&
+      primaryReadonlyMasterKey.exitCodeName === successExitCode
+    );
+  } catch (e) {
+    return false;
```
Comment (cloudify) [inline src/utils/azure.ts]: it could be useful to log the exception to the console, at least it gives you a hint on why the command failed

Comment (Undermaken) [reply src/utils/azure.ts]: I used cli.log instead of cli.error because this one causes the test failure

**Pull request merged**